rizoma/templates/accounts/user_detail.html

```diff
@@ -0,0 +1,177 @@
+                </span>
+            </h5>
+            <div class="card-body">
+                <div class="row">
+                    <div class="col">
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Name" %}:</div>
+                            <div class="col-12 col-sm-8" id="tapir_user_display_name">{% get_display_name_full object %}
+                            </div>
+                        </div>
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Username" %}:</div>
+                            <div class="col-12 col-sm-8" id="tapir_user_username">{{ object.username }}</div>
+                        </div>
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Email" %}:</div>
+                            <div class="col-12 col-sm-8" id="tapir_user_email">{{ object.email }}</div>
+                        </div>
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Phone number" %}:</div>
+                            <div class="col-12 col-sm-8" id="tapir_user_phone_number">
+                                {% if object.phone_number %}
+                                {{ object.phone_number|format_phone_number }}
+                                {% else %}
+                                <span class="text-danger">{% translate "Missing" %}</span>
+                                {% endif %}
+                            </div>
+                        </div>
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Birthdate" %}:</div>
+                            <div class="col-12 col-sm-8" id="tapir_user_birthdate">
+                                {% if object.birthdate %}
+                                {{ object.birthdate|date:"d.m.Y" }}
+                                {% else %}
+                                <span class="text-danger">{% translate "Missing" %}</span>
+                                {% endif %}
+                            </div>
+                        </div>
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Address" %}:</div>
+                            <div class="col-12 col-sm-8" id="tapir_user_address">
+                                {% if object.street and object.city %}
+                                {{ object.get_display_address }}
+                                {% else %}
+                                <span class="text-danger">{% translate "Missing" %}</span>
+                                {% endif %}
+                            </div>
+                        </div>
```

Comment:
You can hide the birthdate and the address: we don't import them from coops.pt so they will always be missing.